Snippets/Snippets_3/BasicUsageOfIBus.cs

```diff
@@ -6,13 +6,31 @@ public class BasicUsageOfIBus
     {
         void Send()
         {
-            IBus bus = null;
-
             #region BasicSend
+            IBus bus = CreateAndStartTheBus();
```

Comment:
in the v5 example you used the real config api to create the IBus instance which I like much better. Any reason to not do the same in v3/4?